src/examples/input-field/input-field-error-icon.tsx

```diff
@@ -0,0 +1,35 @@
+import { Component, h, State } from '@stencil/core';
+const MIN_LENGTH = 6;
+
+@Component({
+    tag: 'limel-example-input-field-error-icon',
+    shadow: true,
+})
+export class InputFieldErrorIconExample {
+    @State()
+    private value;
+
+    constructor() {
+        this.onChange = this.onChange.bind(this);
+    }
+
+    public render() {
+        return (
+            <limel-input-field
+                label="Text Field"
+                minlength={MIN_LENGTH}
+                helperText={this.onHelperText()}
```

Comment:
The function needs to be either renamed or removed. `on` indicates that this is an event or callback which is not the case, and since the text never changes we can just insert the text here instead
```
helperText="..."
```